client/src/components/user/dashboard.js

```diff
@@ -12,10 +13,32 @@ class Dashboard extends React.Component {
     super();
 
     this.state = { 
-      successMessage: ''
+      successMessage: '',
     };
   }
 
+  async componentDidMount() {
+    // When user logs in and Dashboard is loaded, we want to update their subscription due dates.
+    // We also need to know which subscriptions have passed it's due date, in order to notify them that they have late subscriptions.
+    try {
+      const dueDates = await fetch(`${window.location.pathname}/subscriptions/update`);
+
+      const response = await dueDates.json();
+
+      const { lateDueDates } = response;
+      if (lateDueDates.length === 0) {
+        return;
+      }
+
+      for (let i = 0; i < lateDueDates.length; i += 1) {
+        const { name, date } = lateDueDates[i];
+        toast(`Your subscription ${name} was due ${new Date(date).toISOString().substring(0, 10)}`);
```

Comment:
is there a way to make the toast notification sticky and only go away when the user manually closes it? For subscriptions that are either past due or due today, I think the toast notification should stay on the page in case the user were to miss it.

Alternatively/additionally we could visually highlight the subscription cards for those that are either due today or past due with a simple border color difference or something else that stands out to the user